Проверяем рабочий процесс

muxa53 opened the pull request:
---
:mortar_board: [Проверяем рабочий процесс](https://up.htmlacademy.ru/htmlcss/33/user/540991/tasks/2)

:boom: https://htmlacademy-htmlcss.github.io/540991-sedona-33/1/

muxa53 committed:
Добавляет наставника

Danil2005st commented:
Хорошо бы добавить к имени наставника ссылку на профиль, как сделано в имени студента

muxa53 committed:
Добавляет ссылку на профиль наставника

ещё не разобрался есть ли разница но, дополнительно заменил кавычки на квадратные скобки вокруг имени/фамилии
